gui/tests/e2e/swagger.e2e.js

```diff
@@ -28,7 +30,9 @@ describe('Labs Workbench Swagger UI View', function() {
     helpers.afterAll();
   });
   
-  it('should verify page', function() {
-    
+  it('should offer a Workbench API reference', function() {
+    expect(swaggerUiPage.apiTitle.isPresent()).toBe(true);
+    expect(swaggerUiPage.apiTitle.isDisplayed()).toBe(true);
+    expect(swaggerUiPage.apiTitle.getText()).toBe(EXPECTED_API_TITLE);
```

Comment:
`EXPECTED_API_TITLE` is effectively read from the name/description `swagger.yml` spec, so this seems to be a reasonable test of whether or not the SwaggerUI view is actually functioning